src/main/java/com/app/katchup/MeetingResponse/MeetingResponseService.java

```diff
@@ -20,5 +22,15 @@ public MeetingInboxResponse postInboxForUserName(MeetingInboxResponse meetingInb
         meetingResponseRepo.save(meetingInboxResponse);
         return meetingInboxResponse;
     }
+ //update decision for client
+   public Decision putResponseForUserDecision(MeetingResponse meetingUserResponse){
+        MeetingInboxResponse meetingResponse = meetingResponseRepo.findbyUserNameAndMeetingID(meetingUserResponse.getUserName(),
+                meetingUserResponse.getMeetingId());
+        if(meetingUserResponse == null)
+            return null;
+        meetingResponse.setDecision(meetingUserResponse.getDecision());
```

Comment:
Please  add validation for Poll as decision. 
if(Decsioon == Poll){
startDateTime != null && endDateTime != nul & EndDateTime > startDateTime. 
}